src/components/app/createRoutes.jsx

```diff
@@ -20,13 +21,20 @@ export default function createRoutes(routes, authHoc = null) {
     }
 
     // handle private routes (default value, most pages in cms)
+    let component = null;
+    if (route.lazy) {
```

Comment:
@rsumilang does a dev need to set "lazy" on the route?  Ideally they don't and lazy is assumed true unless explicitly set to false.

Reply:
@rsumilang you could change the setting to `eager` or `prefetch` if you want it to read better.

Reply:
@rsumilang let's also pass the props into `route.component(props)` in case we need it in the import function.  and use options to provide our own loader.  @mgollnick will want the admin-ui loader to be used.